LEGO1/mxentity.cpp

```diff
@@ -1,19 +1,15 @@
 #include "mxentity.h"
 
-// OFFSET: LEGO1 0x1001d190 STUB
+// OFFSET: LEGO1 0x1001d190
 MxEntity::MxEntity()
 {
-  // TODO
+  // FIXME: This won't match because of m_atom. How would it access m_internal?
+  this->m_atom = NULL;
```

Comment:
I'm fairly certain MxAtomId is a member of MxEntity (i.e. not a pointer) and `m_internal` being set to NULL is an inlined constructor.

Reply:
so what could this be then? And how could MxAtomId (a class) be a member? 🤔 haven't done class inheritance in a while

Reply:
An object (class) can be a member of another class in C++. Currently you have the pointer to an MxAtomId object (`MxAtomId *`), but the object itself can be embedded by dropping the `*`. The compiler will then call its constructor and destructor automatically without having to write code for it. Then as I said, the setting `m_internal` to `NULL` is probably MxAtomId's inlined constructor.